opencti-platform/opencti-front/src/private/components/events/stix_sighting_relationships/StixSightingRelationshipCreationForm.js

```diff
@@ -242,7 +230,7 @@ const StixSightingRelationshipCreationForm = ({
                   <span className={classes.name}>
                     {isMultipleTo
                       ? (<em>{t_i18n('Multiple entities selected')}</em>)
-                      : (defaultName(toEntity))}
+                      : (getMainRepresentative(toEntity))}
```

Comment:
the value displayed is not troncated to 20 anymore... Problems with long representatives